extra/exports

```diff
@@ -556,3 +407,221 @@ uuid_nil
+lua_tointeger
+lua_tolstring
+lua_tonumber
+lua_topointer
+# <lua_tostring> is a macro to <lua_tolstring>.
+lua_tothread
+lua_touserdata
+lua_type
+lua_typename
+# <lua_upvalueindex> is a macro to LUA_GLOBALSINDEX.
+lua_xmove
+lua_yield
+
+# Auxiliary interfaces from Lua 5.1 Reference Manual (i.e. Lua
+# auxiliary library functions from lauxlib.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+# <luaL_addchar> is a macro.
+luaL_addlstring
+# <luaL_addsize> is a macro.
+luaL_addstring
+luaL_addvalue
+# <luaL_argcheck> is a macro to <luaL_argerror>.
+luaL_argerror
+luaL_buffinit
+luaL_callmeta
+luaL_checkany
+# <luaL_checkint> is a macro to <luaL_checkinteger>.
+luaL_checkinteger
+# <luaL_checklong> is a macro to <luaL_checkinteger>.
+luaL_checklstring
+luaL_checknumber
+luaL_checkoption
+luaL_checkstack
+# <luaL_checkstring> is a macro to <luaL_checklstring>.
+luaL_checktype
+luaL_checkudata
+# <luaL_dofile> is a macro to <luaL_loadfile> + <lua_pcall>.
+# <luaL_dostring> is a macro to <luaL_loadstring> + <lua_pcall>.
+luaL_error
+luaL_getmetafield
+# <luaL_getmetatable> is a macro to <lua_getfield> + LUA_REGISTRYINDEX.
+luaL_gsub
+luaL_loadbuffer
+luaL_loadfile
+luaL_loadstring
+luaL_newmetatable
+luaL_newstate
+luaL_openlibs
+# <luaL_optint> is a macro to <luaL_optinteger>.
+luaL_optinteger
+# <luaL_optlong> is a macro to <luaL_optinteger>.
+luaL_optlstring
+luaL_optnumber
+# <luaL_optstring> is a macro to <luaL_optlstring>.
+luaL_prepbuffer
+luaL_pushresult
+luaL_ref
+luaL_register
+# <luaL_typename> is a macro to <lua_typename>.
+luaL_typerror
+luaL_unref
+luaL_where
+
+# Interfaces from Lua 5.2 Reference Manual (i.e. Lua C API
+# functions from lua.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+lua_copy
+lua_loadx
+lua_tointegerx
+lua_tonumberx
+lua_upvalueid
+lua_upvaluejoin
+lua_version
+
+# Auxiliary interfaces from Lua 5.2 Reference Manual (i.e. Lua
+# auxiliary library functions from lauxlib.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+luaL_execresult
+luaL_fileresult
+luaL_loadbufferx
+luaL_loadfilex
+# <luaL_newlibtable> is a macro to <lua_createtable>.
+# <luaL_newlib> is a macro to <luaL_newlibtable> + <luaL_setfuncs>.
+luaL_setfuncs
+luaL_setmetatable
+luaL_testudata
+luaL_traceback
+
+# Interfaces from Lua 5.3 Reference Manual (i.e. Lua C API
+# functions from lua.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+lua_isyieldable
+# Auxiliary interfaces from Lua 5.3 Reference Manual (i.e. Lua
+# auxiliary library functions from lauxlib.h).
+# XXX: Interfaces implemented via C macros are also listed below
+# only for consistency.
+# <luaL_opt> is a macro to <lua_isnoneornil>.
+
+# Auxiliary interfaces provided by LuaJIT (i.e. public
+# functions from lauxlib.h, which are not listed in any Lua
+# Reference Manual).
+luaL_findtable
```

Comment:
However, it anyway occasionally used in modules ([example](https://github.com/openresty/stream-lua-nginx-module/blob/309198abf26266f1a3e53c71388ed7bb9d1e5ea2/src/ngx_stream_lua_clfactory.c#L677)), because it works and convenient. I don't think that we would make anybody happy by hiding it, but we can make someone sad.

Reply:
Agree with @Totktonada here.